docs/sources/tempo/release-notes/v2-7.md

```diff
@@ -215,7 +215,7 @@ This release disables gRPC compression in the querier and distributor for perfor
 Our benchmark suggests that without compression, queriers and distributors use less CPU and memory.
 
 {{< admonition type="note" >}}
-While disabling gRPC compression improves performance, it will increase data usage and network traffic, which can impact cloud billing depending on your configuration.
+While disabling gRPC compression improves performance, it may increase data usage and network traffic, which can impact Grafana Cloud billing, especially for larger clusters.
```

Comment:
```suggestion
While disabling gRPC compression improves performance, it will increase data usage and network traffic, which can impact cloud billing, especially for larger clusters.
```